packages/trader/src/Modules/Account/Containers/account.jsx

```diff
@@ -80,9 +83,9 @@ class Account extends React.Component {
 
                 if (
                     !needs_verification &&
```

Comment:
can remove this needs_verification for allowing client to go to authentication page, only allow_document_upload is needed

Reply:
as discussed, will remove the `needs_verification` check as well